pandas/tests/io/parser/data/items.jsonl

```diff
@@ -0,0 +1,2 @@
+{"a": 1, "b": 2}
```

Comment:
what is the purpose of this file?

Reply:
I see, ok you have to have this named ``.json`` otherwise it won't be picked up by ``setup.py`` (IOW the install test will fail).